### PR TITLE
Replace README with deprecation notice

### PR DESCRIPTION
**Pull request by gfontenot**
We're no longer actively maintaining this library, so we should make that
crystal clear for people that happen across this repo.

### CODE REVIEW

**Comment by gfontenot** [comment]
We'll also archive this repo as soon as I can figure out who has those specific superpowers.

**Comment by brianpartridge** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,87 +1,5 @@
...
-* 1.0 – Swift 3.0, Xcode 8+, and iOS 10 SDK adoption. Currently in development.
-
-## Contributing
-
-We’re glad you’re interested in SuperDelegate, and we’d love to see where you take it. Please read our [contributing guidelines](Contributing.md) prior to submitting a Pull Request.
-
-Thanks, and happy delegating!
+This library has been deprecated and is no longer under active development.
+The original README can be found in [README_OLD](README_OLD.md), but we
+would recommend moving away from using this library in the future.
```
Comment (dfed) [inline README.md]: Should we include a section asking for new maintainers? If someone wants to maintain this, I'd be in favor of transferring ownership.

**Comment by gfontenot** [comment]
Not sure how to really summarize the reasons we've moved away from using this, but I can definitely add a call to action for people that would like to take over ownership.

**Pull request merged**